lib/manager/pip_setup/extract.js

```diff
@@ -40,7 +62,13 @@ async function extractSetupFile(content, packageFile, config) {
     );
   } else {
     logger.info('Running python via global command');
-    cmd = 'python';
+    const alias = await renovateCache.get(cacheNamespace, 'pythonAlias');
+    if (alias) {
+      cmd = alias;
+    } else {
+      cmd = await getPythonAlias();
+      await renovateCache.set(cacheNamespace, 'pythonAlias', cmd);
```

Comment:
I think we can cache this for a long time, perhaps a day? Default ttl if unspecified is 5 minutes.